website/archiver/model.py

```diff
@@ -0,0 +1,151 @@
+import datetime
+
+from modularodm import fields
+
+from framework.mongo import ObjectId
+from framework.mongo import StoredObject
+
+from website.archiver import (
+    ARCHIVER_INITIATED,
+    ARCHIVER_SUCCESS,
+    ARCHIVER_FAILURE,
+    ARCHIVER_FAILURE_STATUSES
+)
+
+from website.addons.base import StorageAddonBase
+from website import settings
+
+class ArchiveTarget(StoredObject):
+
+    _id = fields.StringField(
+        primary=True,
+        default=lambda: str(ObjectId())
+    )
+
+    name = fields.StringField()
```

Comment:
@samchrisinger Can you document what this field is? What are example values? Also, is this field required?